atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleQueryFactory.java

```diff
@@ -412,7 +413,50 @@ private String wrapQueryWithIncludeValue(String wrappedName, String query, boole
                 + "   AND wrap.ts = i.ts ";
     }
 
-    abstract String getValueSubselect(String tableAlias, boolean includeValue);
+    private String getValueSubselect(String tableAlias, boolean includeValue) {
+        if (includeValue) {
+            List<String> colNames = getValueColumnNames();
+            StringBuilder ret = new StringBuilder(10 * colNames.size());
+            for (String colName : colNames) {
+                // e.g., ", m.val"
+                ret.append(", ").append(tableAlias).append('.').append(colName);
+            }
+            return ret.toString();
+        } else {
+            return "";
```

Comment:
The old getValueSubselect returned a space in this case instead of empty string. Does that cause any issues?